okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -42,22 +45,31 @@
 import okhttp3.mockwebserver.SocketPolicy;
 import okio.Buffer;
 import okio.BufferedSink;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.CoreMatchers;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 
 import static java.util.Arrays.asList;
 import static okhttp3.TestUtil.defaultClient;
+import static org.hamcrest.CoreMatchers.any;
```

Comment:
I’m not 100% sure we have this dependency when OkHttp is consumed by AOSP

Reply:
Should we find out first? or land this PR and then revert the dependency on hamcrest if needed?

Reply:
It's a core dependency of junit 4.12, so it would surprise me if it runs without

http://mvnrepository.com/artifact/junit/junit/4.12

Reply:
Yep, it’s there. Whew. https://android.googlesource.com/platform/external/hamcrest/